packages/odyssey-react-mui/src/Status.tsx

```diff
@@ -36,14 +36,13 @@ export type StatusProps = {
    * The style of the Status indicator
    */
   variant?: (typeof statusVariantValues)[number];
+  /**
+   * @deprecated All Statuses are now `pill` variant.
+   */
+  /** **Deprecated:** All Statuses are now `pill` variant */
```

Comment:
Curious if you meant to put this comment block after the props and just above the component variable declaration below?

Reply:
Whoops, yeah, this needs to be above `variant`. I'll fix. Thanks!